cursor_pr_review.py

```
@@ -233,7 +273,233 @@ def _validate_anthropic(self) -> Dict[str, Any]:
+            response = self.session.get(
+                f"https://api.github.com/repos/{repo}/pulls/{pr_number}",
+                headers={
+                    "Authorization": f"token {self.config.github_token}",
+                    "Accept": "application/vnd.github.v3.diff"
+                }
+            )
+            response.raise_for_status()
+            return response.text
+
+        except requests.exceptions.RequestException as e:
+            raise APIError(f"Failed to get PR diff: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)